nusacrowd/nusa_datasets/su_emot/su_emot.py

```diff
@@ -0,0 +1,131 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from nusacrowd.utils import schemas
+from nusacrowd.utils.configs import NusantaraConfig
+from nusacrowd.utils.constants import DEFAULT_NUSANTARA_VIEW_NAME, DEFAULT_SOURCE_VIEW_NAME, Tasks
+
+_DATASETNAME = "su_emot"
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_NUSANTARA_VIEW_NAME
+
+_LANGUAGES = ["sun"]
+_LOCAL = False
```

Comment:
Hi @nadyadtm, one minor comment again. Since this dataset is local language, you can set this to `True` ya. Thanks!

Reply:
@muhsatrio I think we use `_LOCAL` to mark if the dataset has to be downloaded manually prior to using the dataloader, hence it should be `False`.

Reply:
ah I see, I think it's related to language of dataset whether it's Indonesian or local language. Okay, I think this comment can be resolved, thank you for the information @holylovenia!